File: 2535.py
nums = list(map(int, input().split()))
elsum = sum(nums)
digsum = 0

for i in range(len(nums)):
    while nums[i] > 0:
        digsum+= nums[i]%10
        nums[i]//=10

print(abs(elsum - digsum))

File: 2535.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    int elsum = 0;
    int digsum = 0;
    cin >> n;
    int arr[n];
    for (int i = 0; i < n; i++) {
        cin >> arr[i];
        elsum+= arr[i];
    }

    for (int i = 0; i < n; i++) {
        while (arr[i] > 0) {
            digsum += arr[i] % 10;
            arr[i] /= 10;
        }
    }

    cout << abs(elsum - digsum);
    
}